Sources/SmileID/Classes/Networking/Models/MultiPart.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+struct MultiPartMedia: Encodable {
+    let key: String
+    let filename: String
+    let data: Data
+    let mimeType: String
+
+    init?(withImage image: Data, forKey key: String, forName name: String) {
+        self.key = key
+        mimeType = "image/jpeg"
+        filename = name
+        data = image
+    }
+}
+
+public struct MultiPartRequest: Encodable {
```

Comment:
Can you call this SmartSelfieRequest? Multipart is a networking concept and the request is an application level concept